rust/core-lib/src/tabs.rs

```diff
@@ -364,9 +360,12 @@ impl<W: Write + Send + 'static> Documents<W> {
         self.buffers.lock().editor_for_view_mut(view_id).unwrap().do_rpc(view_id, cmd)
     }
 
+    #[allow(unused_variables)]
```

Comment:
Yea I had it that way first, but these are going to become used in imminent future work. I guess I've generally been using `#[allow()]` mostly to annotate sections of code that are unimplemented? This might not be very idiomatic / great practice, generally.